projects/plugins/boost/app/modules/image-size-analysis/data-sync/class-image-analysis-action-fix.php

```diff
@@ -45,7 +45,7 @@ public function handle( $data, $_request ) {
 			$changed                    = 'fix';
 		}
 
-		if ( $changed ) {
+		if ( 'fix' === $changed || 'removed' === $changed ) {
```

Comment:
🤷 Looks like it's always going to be one of these.

Reply:
Yep, it's a bit silly. I opted for a more explicit check to prevent unexpected behavior if there's another string added, but it's another case I'd've prefer a bool flag rather than a string.